examples/evaluate_a_judge_model_capabilities_on_arena_hard.py

```diff
@@ -0,0 +1,37 @@
+from unitxt import evaluate
+from unitxt.inference import (
+    HFPipelineBasedInferenceEngine,
+)
+from unitxt.standard import StandardRecipe
+
+model_id = "meta-llama/llama-3-70b-instruct"
+model_format = "formats.llama3_instruct"
+
+dataset = (
+    StandardRecipe(
```

Comment:
We should use load_dataset() and not StandardRecipe , which is an internal implementation.

Reply:
load_dataset(card="cards.arena_hard.response_assessment.pairwise_comparative_rating.both_games_gpt_4_judge, template="templates.response_assessment.pairwise_comparative_rating.arena_hard_with_shuffling".format=model_format,loader_limit=4)

Reply:
@yoavkatz we cannot use loader_limit=4, because there is a join operation in the card. We have to load at least 10k examples to not get an error